Apps/CocoaPods-FCM/src/View/SettingsView.swift

```diff
@@ -1,60 +1,93 @@
 import CioTracking
 import SampleAppsCommon
 import SwiftUI
+import UIKit
 
 struct SettingsView: View {
-    var done: () -> Void
+    var siteId: String?
+    var apiKey: String?
 
-    private let settingsManager = CioSettingsManager()
+    var done: () -> Void
 
     @StateObject private var viewModel = ViewModel()
 
     var body: some View {
-        // TODO: add back button in UI to dismiss screen. add appium ID to it.
-
-        VStack(spacing: 5) {
-            SettingsTextField(title: "Tracking URL:", value: $viewModel.settings.trackUrl).setAppiumId("Track URL Input")
-            SettingsTextField(title: "Site id:", value: $viewModel.settings.siteId).setAppiumId("Site ID Input")
-            SettingsTextField(title: "API key:", value: $viewModel.settings.apiKey).setAppiumId("API Key Input")
-            SettingsTextField(title: "BQ seconds delay:", value: $viewModel.settings.bqSecondsDelay.toStringBinding())
-            SettingsTextField(title: "BQ min number tasks:", value: $viewModel.settings.bqMinNumberTasks.toStringBinding())
-            SettingsToggle(title: "Track screens", isOn: $viewModel.settings.trackScreens).setAppiumId("Track Screens Toggle")
-            SettingsToggle(title: "Track device attributes", isOn: $viewModel.settings.trackDeviceAttributes).setAppiumId("Track Device Attributes Toggle")
-            SettingsToggle(title: "Debug mode", isOn: $viewModel.settings.debugSdkMode).setAppiumId("Debug Mode Toggle")
-
-            ColorButton(title: "Save") {
-                // save settings to device storage for app to re-use when app is restarted
-                settingsManager.settings = viewModel.settings
-
-                // Re-initialize the SDK to make the config changes go into place immediately
-                CustomerIO.initialize(siteId: viewModel.settings.siteId, apiKey: viewModel.settings.apiKey, region: .US) { config in
-                    viewModel.settings.configureCioSdk(config: &config)
+        ZStack {
+            BackButton {
+                done()
+            }
+
+            VStack(spacing: 5) {
+                HStack {
+                    Text("Device token: ")
+                    OneLineText(viewModel.pushToken)
+                    Button(action: {
+                        UIPasteboard.general.string = viewModel.pushToken
+                    }) {
+                        Image(systemName: "list.clipboard.fill").font(.system(size: 24))
+                    }
                 }
 
-                done()
-            }.setAppiumId("Save Settings Button")
+                Group {
```

Comment:
If I was using the Groups to connect similar views, I think I would have created a few groups here. This might just be something to learn with swift UI.